src/components/cart/CartSummary.jsx

```diff
@@ -0,0 +1,58 @@
+import { Button, Typography } from 'antd';
+import { useCartStore } from '../../store';
+import { handleFormatAmountToPHP } from '../../utils';
+
+const CartSummary = () => {
+	const { itemsInCart } = useCartStore((state) => state);
+
+	const totalQuantity = itemsInCart.reduce((prev, next) => {
+		return prev + next.items[0].quantity;
+	}, 0);
+
+	var totalPrice = itemsInCart.reduce((prev, next) => {
+		return prev + next.items[0].total_price;
+	}, 0);
+
+	totalPrice = handleFormatAmountToPHP(totalPrice);
```

Comment:
remove this. do the formatting directly in line 25 and 46

Reply:
done.